### ISSUE TITLE
What are .npy files in Image recognition model

### ISSUE THREAD

**Issue by jitendersaini** [afshinea/keras-data-generator#2]
Hi, Thanks for such a nice post. I'm a new learner and have few doubts here. I want to train my model from a big dataset of images those are classified in folder names like red_floral_skirts/1.jpeg,2.jpeg... n.jpeg and black_striped_shirts/1.jpeg,2.jpeg... n.jpeg. Now I want to predict from my model like black: 98%, striped:97%, shirt:98% etc. (My model should tell that this is a black, floral and shirt) now my training data should be the collection of images and labels should be like ['black','striped','shirt'] for all images under folder black_striped_shirts. I really need your help here to know that how can I fit my requirement in this DataGenerator. Do I need to replace partition dictionary from id_1 to actual images and label dictionary with folder names with splitting into 3? and in below code snippet what is data/ID_1.npy file? from where .npy file comes in my case?

for i, ID in enumerate(list_IDs_temp):
# Store sample
X[i,] = np.load('data/' + ID + '.npy')

Thanks a lot for your help,
Jitender
